app/default/tags.conf

```diff
@@ -0,0 +1,3 @@
+[sourcetype=heroku_requests]
+web = enabled
+proxy = enabled
```

Comment:
This marks our heroku request messages as compatible with the web CIM. As a result they should be available to view in the Web Dataset in Splunk once the indexes are added to the `cim_Web_indexes` macro.